moving_zeroes/moving_zeroes.py

```diff
@@ -3,9 +3,23 @@
 Returns: a List of integers
 '''
 def moving_zeroes(arr):
-    # Your code here
-
-    pass
+    
+    return [x for x in arr if x != 0] + [0] * arr.count(0)
+    
```

Comment:
My jaw literally hit the floor when I read this implementation.

This is _stellar_ work.